src/main/java/com/amazon/aws/iot/greengrass/configuration/common/Configuration.java

```diff
@@ -29,7 +29,7 @@ public class Configuration {
 
     public static final FailureHandlingPolicy DEFAULT_FAILURE_HANDLING_POLICY = FailureHandlingPolicy.ROLLBACK;
 
-    public static final SchemaDate DEFAULT_SCHEMA_DATE = SchemaDate.MAY_17_2021;
+    public static final SchemaDate DEFAULT_SCHEMA_DATE = SchemaDate.DEC_02_2022;
```

Comment:
I don't think you should be changing this, I'd expect this to cause things to fail as being incompatible. Adding extra fields is safe.

Reply:
Makes sense I will revert the schema date portion.